requirements.txt

```diff
@@ -70,6 +70,7 @@ pyasn1==0.4.8
 pychalk==2.0.1
 pycparser==2.20; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
 pycryptodome==3.9.9
+pycryptodomex==3.10.1
```

Comment:
I'm wondering if there can be any implication of having both `pycryptodome` and `pycryptodomex`. It seems the x version was released to avoid interference with the pycrypto package, which we don't use anyway (https://www.pycryptodome.org/en/latest/src/introduction.html?highlight=pycryptodomex#pycryptodome). It seems in our case we can potentially use both, but probably it's a dependency issue?